src/app/screens/GenerateAccount/GenerateAccountVerify.tsx

```diff
@@ -41,45 +51,49 @@ export const GenerateAccountVerify = ({ onNext }: IFlowComponentProps) => {
-      <form onSubmit={handleSubmit}>
-        <Box mt="3">
-          <Label htmlFor="firstWord">
-            <Body>
-              {translate('WHAT_IS_NTH_WORD', { $nth: translateRaw(`ORDINAL_${first + 1}`) })}
-            </Body>
-          </Label>
-          <FormInput id="firstWord" name="firstWord" data-testid="firstWord" form={form} />
-          <FormError name="firstWord" form={form} />
+    <>
+      <Container pt="0">
+        <Box sx={{ textAlign: 'center' }} mb="4">
+          <Heading fontSize="24px" lineHeight="150%" mb="2">
+            {translateRaw('VERIFY_MNEMONIC_PHRASE_TITLE')}
+          </Heading>
+          <Body>{translate('VERIFY_MNEMONIC_PHRASE_DESCRIPTION')}</Body>
         </Box>
-        <Box mt="3">
-          <Label htmlFor="secondWord">
-            <Body>
-              {translate('WHAT_IS_NTH_WORD', { $nth: translateRaw(`ORDINAL_${second + 1}`) })}
-            </Body>
-          </Label>
-          <FormInput id="secondWord" name="secondWord" data-testid="secondWord" form={form} />
-          <FormError name="secondWord" form={form} />
-        </Box>
-        <Box mt="3">
-          <Label htmlFor="thirdWord">
-            <Body>
-              {translate('WHAT_IS_NTH_WORD', { $nth: translateRaw(`ORDINAL_${third + 1}`) })}
-            </Body>
-          </Label>
-          <FormInput id="thirdWord" name="thirdWord" data-testid="thirdWord" form={form} />
-          <FormError name="thirdWord" form={form} />
-        </Box>
-        <Button mt="4" type="submit" mb="3">
+        <form onSubmit={handleSubmit} id="mnemonic-phrase-verify-form">
+          <Box mt="3">
+            <Label htmlFor="firstWord">
+              <Body>
+                {translate('WHAT_IS_NTH_WORD', { $nth: translateRaw(`ORDINAL_${first + 1}`) })}
+              </Body>
+            </Label>
+            <FormInput id="firstWord" name="firstWord" data-testid="firstWord" form={form} />
+            <FormError name="firstWord" form={form} />
+          </Box>
+          <Box mt="3">
+            <Label htmlFor="secondWord">
+              <Body>
+                {translate('WHAT_IS_NTH_WORD', { $nth: translateRaw(`ORDINAL_${second + 1}`) })}
+              </Body>
+            </Label>
+            <FormInput id="secondWord" name="secondWord" data-testid="secondWord" form={form} />
+            <FormError name="secondWord" form={form} />
+          </Box>
+          <Box mt="3">
+            <Label htmlFor="thirdWord">
+              <Body>
+                {translate('WHAT_IS_NTH_WORD', { $nth: translateRaw(`ORDINAL_${third + 1}`) })}
+              </Body>
+            </Label>
+            <FormInput id="thirdWord" name="thirdWord" data-testid="thirdWord" form={form} />
+            <FormError name="thirdWord" form={form} />
+          </Box>
+        </form>
+      </Container>
+      <PanelBottom variant="clear">
+        <Button type="submit" form="mnemonic-phrase-verify-form">
           {translateRaw('VERIFY_WORDS')}
```

Comment:
This button looks off - not centered at the bottom. Is this a pattern we want to use instead of the ones with the white background? Those look better imo